codecov.yml

```diff
@@ -0,0 +1,10 @@
+coverage:
+  status:
+    project:
+      default:
+        target: 85%
```

Comment:
@najuzilu Since this is a relatively smaller project, should we keep the target of the whole project to be a bit higher? like 90 or above?
what do you think @mmcky

Reply:
Let's keep it at 85% that matches other `ebp` projects.